test/functional/apps/management/_import_objects.js

```diff
@@ -280,6 +282,54 @@ export default function ({ getService, getPageObjects }) {
         expect(isSuccessful).to.be(true);
       });
 
+      it('should allow the user to confirm overriding multiple duplicate saved objects', async function () {
+        // This data has already been loaded by the "visualize" esArchive. We'll load it again
+        // so that we can override the existing visualization.
+        await PageObjects.savedObjects.importFile(
+          path.join(__dirname, 'exports', '_import_objects_multiple_exists.json'),
+          false
+        );
+
+        await PageObjects.savedObjects.checkImportLegacyWarning();
+        await PageObjects.savedObjects.checkImportConflictsWarning();
+
+        await PageObjects.settings.associateIndexPattern('logstash-*', 'logstash-*');
+        await PageObjects.savedObjects.clickConfirmChanges();
+
+        // Override the visualizations.
+        await PageObjects.common.clickConfirmOnModal(false);
+        // as the second confirm can pop instantly, we can't wait for it to be hidden
+        // with is why we call clickConfirmOnModal with ensureHidden: false in previous statement
+        // but as the initial popin can take a few ms before fading, we need to wait a little
+        // to avoid clicking twice on the same modal.
+        await delay(1000);
+        await PageObjects.common.clickConfirmOnModal(false);
+
+        const isSuccessful = await testSubjects.exists('importSavedObjectsSuccess');
```

Comment:
optional: can be combined with an assertion as `await testSubjects.exists('importSavedObjectsSuccess');`